components/Shared/TopNav.tsx

```diff
@@ -0,0 +1,73 @@
+import Logo from "@/public/assets/icons/dashboard/Logo.svg";
+import {
+	ArrowDownIcon,
+	MenuIcon,
+	NotificationIcon,
+	PlusButtonIcon,
+} from "@/public/assets/icons/dashboard/icons";
+import userIcon from "@/public/assets/icons/dashboard/userIcon.svg";
+import Image from "next/image";
+import Link from "next/link";
+import { Button } from "../ui";
+
+type TopNavProps = {
+	toggleSidebar: () => void;
+};
+
+export default function TopNav({ toggleSidebar }: TopNavProps) {
+	return (
+		<div className="sticky top-0 flex items-center justify-between border-b border-gray-400 p-7  md:border-gray-300 md:p-4 md:pr-10">
+			<div className="hidden md:block">
+				<h1 className="text-sm font-semibold"> Hi, Locs Designer👋</h1>
+				<p className="text-sm">Here is an overview of your campaign ✨</p>
+			</div>
+			<div className="flex items-center justify-center gap-3 md:hidden">
+				<div className="cursor-pointer" onClick={toggleSidebar}>
```

Comment:
i don't understand why you adding onClick to a div when you can use the button component?